app/src/backend/environment/hooks/useEnvironmentManager.ts

```diff
@@ -215,6 +215,27 @@ const useEnvironmentManager = (initListenerAndFetcher: boolean = false) => {
     [allEnvironmentData]
   );
 
+  const getVariableData = useCallback(
+    (variableName: string) => {
```

Comment:
Why are we getting the variable details using name why not its id?

Reply:
We currently don't maintain `id` for variables only for envs